test/Feature/SeedConcretizeExternalCall.c

```diff
@@ -0,0 +1,26 @@
+// RUN: %clang -emit-llvm -c -g %s -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out --entry-point=TestGen %t.bc "initial"
+// RUN: test -f %t.klee-out/test000001.ktest
+// RUN: not test -f %t.klee-out/test000002.ktest
+
+// RUN: rm -rf %t.klee-out-2
+// RUN: %klee --external-calls=all --output-dir=%t.klee-out-2 --seed-file %t.klee-out/test000001.ktest %t.bc
```

Comment:
I would structure this test as follows:. 
1) run KLEE here with `-exit-on-error`, 2) remove the `RUN: grep` line and 3) change the assert into an equality, i.e., `assert(abs(z) == 0xBADA55)`

Reply:
Also add a comment at the beginning of the test saying what this is testing, e.g., "Testing that seeds are used during concretization of arguments in external calls".